nids.rst

```diff
@@ -29,6 +29,44 @@ To tune the detection:
 .. image:: images/60_detection_nids_2_tuning_2_add.png
   :target: _images/60_detection_nids_2_tuning_2_add.png
 
+Enabling | Disabling with Regex
+---------------------
```

Comment:
Please make sure that section underlines are the same length as the title. Otherwise, sphinx will generate warnings.